src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -13,7 +13,7 @@
     <Title>Confluent.Kafka</Title>
     <AssemblyName>Confluent.Kafka</AssemblyName>
     <VersionPrefix>0.11.3</VersionPrefix>
-    <TargetFrameworks>net45;net46;netstandard1.3;</TargetFrameworks>
+    <TargetFrameworks>net45;net46;netstandard1.3;netstandard2.0;</TargetFrameworks>
```

Comment:
adding this here is isn't necessary - we require both net45/net46 since we have different code paths for each platform, but we don't make use of any netstandard2.0 features.